PyPDF2/merger.py

```diff
@@ -123,11 +124,15 @@ def merge(self, position, fileobj, bookmark=None, pages=None, import_bookmarks=T
             filecontent = StreamIO(fileobj.stream.read())
             fileobj.stream.seek(orig_tell) # reset the stream to its original location
             fileobj = filecontent
+            if hasattr(fileobj, '_decryption_key'):
+                decryption_key = fileobj._decryption_key
```

Comment:
I think this should be above line 126, since StreamIO won't have`_decription_key`

Reply:
Approve this ! put these two lines before 126 make it works !

Thanks for the note. (PyPDF2==1.26.0)